BitwardenShared/Core/Vault/Services/Fido2CredentialStoreService.swift

```diff
@@ -81,3 +104,44 @@ private extension Cipher {
             && login?.fido2Credentials?.isEmpty == false
     }
 }
+
+/// A wrapper of a `Fido2CredentialStore` which adds debugging info for the `Fido2DebugginReportBuilder`.
+class DebuggingFido2CredentialStoreService: Fido2CredentialStore {
```

Comment:
🎨 If we only use this on debug builds, should we gate all of it behind an `#if DEBUG`?